src/Shared/Cosmos/CosmosConstants.cs

```diff
@@ -5,6 +5,12 @@ namespace Aspire.Hosting.Azure.Cosmos;
 
 internal static class CosmosConstants
 {
+    /// <summary>
+    /// Defines the application name used to interact with the Azure cosmos db. This will be suffixed to the
```

Comment:
I'm fine with this, but I think for master we should either change it to:

> the Azure Cosmos database

or

> Azure Cosmos DB

Have "the Azure cosmos db" is strange.